src/engine/Engine.cpp

```diff
@@ -1129,6 +1129,8 @@ void Engine::performMILPSolverBoundedTightening()
         case GlobalConfiguration::MILP_ENCODING_INCREMENTAL:
             _networkLevelReasoner->MILPPropagation();
             break;
+        case GlobalConfiguration::NONE:
+            break;
```

Comment:
I suggest to have a "return" instead of a "break", to save time on the following instructions